apps/okreads/browser/src/app/app.component.html

```diff
@@ -4,7 +4,7 @@
       <div class="header--content">
         <h1>okreads</h1>
         <div>
-          <button
+          <button aria-label="Reading List"
```

Comment:
This label isn't needed, since the text contained ("Reading List") is already an appropriate label.